scripts/create_newcase

```diff
@@ -188,12 +201,11 @@ sub _create_caseroot_tools
 }
 
 
-
 sub _check_input_options
 {
     my ($Case,$opts) = @_;
     
-    # Values in uppercase here have a direct correspondence to Entries in the case XML files
+    # Values in uppercase here have a direct correspondence to Entries in the caseroot XML files
```

Comment:
I thought that case was a bit ambiguous. Its actually the files in the
caseroot directory.

On Sun, Nov 29, 2015 at 6:28 PM, jedwards4b <notifications@github.com>
wrote:

> In scripts/create_newcase
> <https://github.com/jedwards4b/cime/pull/1#discussion_r46102892>:
>
> >  sub _check_input_options
> >  {
> >      my ($Case,$opts) = @_;
> >
> > -    # Values in uppercase here have a direct correspondence to Entries in the case XML files
> > +    # Values in uppercase here have a direct correspondence to Entries in the caseroot XML files
>
> I don't understand this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jedwards4b/cime/pull/1/files#r46102892>.
>